lib/util/ajv.js

```diff
@@ -17,7 +17,7 @@ const Ajv = require("ajv"),
 
 const ajv = new Ajv({
     meta: false,
-    validateSchema: false,
+    validateSchema: true,
```

Comment:
Is this a breaking change? It seems like it will start reporting errors for schemas that aren't "valid" (e.g. they have extra properties) even though they would still work properly.

(However, if `RuleTester` has been throwing an error for that case all along then it might reduce the chance that anyone is relying on the behavior.)

Reply:
emm... I guess so.

and seems it also has a perf loss.
before:
![image](https://user-images.githubusercontent.com/13050025/43673633-fdcf19b2-97f8-11e8-9d6b-daaa818de91f.png)

after:
![image](https://user-images.githubusercontent.com/13050025/43673636-07b2d626-97f9-11e8-9cb5-e1a32e2cbaf5.png)

Reply:
sounds good! should open another PR?

Reply:
You can open another one if you'd like -- alternatively, I think it would be fine to modify this one.